src/wp-includes/blocks.php

```diff
@@ -646,6 +646,13 @@ function _excerpt_render_inner_columns_blocks( $columns, $allowed_blocks ) {
 	return $output;
 }
 
+global $current_parsed_block;
```

Comment:
I'm not sure where a global's initial value should be set. It looks like we set a bunch in `src/wp-settings.php`, so maybe that could work? cc. @SergeyBiryukov

Reply:
At any rate it needs a doc comment describing what the global is and a `@type array`.

Reply:
What is it, actually? @youknowriad ?

Reply:
The current block being rendered, I'm not sure this need initialization at all.

Reply:
If this stays, the `@global $current_arsed_blocks` entry needs to be added to the `render_block()` function below.